lib/generators/web_app_manifest_initializer.txt

```diff
@@ -6,5 +6,10 @@
 # to asset_path in the _favicon.html.erb partial.
 
 Rails.application.config.assets.configure do |env|
-  env.register_mime_type('application/manifest+json', extensions: ['.webmanifest'])
+  if(Sprockets::VERSION.split('.')[0].to_i >= 4)
```

Comment:
No need for the parenthesis, and Ruby usually favors spaces around conditions:

```suggestion
  if Sprockets::VERSION.split('.')[0].to_i >= 4
```

Also works:

```ruby
if Sprockets::VERSION.to_i >= 4
```

Also, do you know if there's a more specific way to determine why adding the `.erb` extension is needed, beyond the version itself? Maybe a Sprocket internal method or feature flag?

Reply:
At least i am not aware of any flags indicting that the processors are not chained automatically anymore ([see here](https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#adding-erb-support-to-your-extension)). They just don't support this chaining by file ending anymore.

Reply:
Got it. In this case, here's the same approach but with a bit less repetition, which I think highlights the different behaviors between Sprockets version a bit more:

```ruby
mime_type = 'application/manifest+json'
extensions = ['.webmanifest']

if Sprockets::VERSION.to_i >= 4
  extensions << '.webmanifest.erb'
  env.register_preprocessor(mime_type, Sprockets::ERBProcessor)
end

env.register_mime_type(mime_type, extensions: extensions)
```

What do you think?

Regardless, your PR works, is readable and should go in :) Thanks for the fix

Reply:
Thanks for the review. I adapted the initializer code. 

Nevertheless, the fix will only work for newly installed rails_real_favcon installations as the existing files aren't patched. Maybe there should be another hint of how to handle sprockets 4 within the readme or changelog when a new version is published.